packages/client/src/components/Router.svelte

```diff
@@ -20,6 +23,55 @@
   // Keep query params up to date
   $: routeStore.actions.setQueryParams(parseQueryString($querystring))
 
+  // Get settings for the root layout
+  $: screen = $screenStore.activeScreen
+  $: embedded = $appStore.embedded
+  $: application = $appStore.application
+  $: navigation = $builderStore.navigation
+  $: logoUrl = $orgStore?.logoUrl
+  $: layoutSettings = getLayoutSettings(
+    screen,
+    embedded,
+    application,
+    navigation,
+    logoUrl
+  )
+
+  const getLayoutSettings = (screen, embedded, app, navigation, logoUrl) => {
```

Comment:
This is the only real addition in this PR. Logic for how the layout component is rendered has moved from the screen store into the Router here.